ios/BT/ExposureManager.swift

```diff
@@ -536,10 +440,103 @@ private extension ExposureManager {
     }
   }
 
-  func cleanup() {
-    // Delete downloaded files from file system
-    localUncompressedURLs.cleanup()
-    localUncompressedURLs = []
-    downloadedPackages = []
+  // MARK: == Exposure Detection Private Promisses ==
```

Comment:
~Promisses~ Promises

Reply:
what about naming this function `fetchIndexFile` (because it is fetching the index file from the server)?

Reply:
yes, I'll change it